<!-- 
RSS generated by JIRA (6.3.4#6332-sha1:51bc225ef474afe3128b2f66878477f322397b16) at Sun May 17 03:38:46 UTC 2015

It is possible to restrict the fields that are returned in this document by specifying the 'field' parameter in your request.
For example, to request only the issue key and summary add field=key&field=summary to the URL of your request.
For example:
https://issues.apache.org/jira/si/jira.issueviews:issue-xml/DERBY-6058/DERBY-6058.xml?field=key&amp;field=summary
-->
<rss version="0.92" >
<channel>
    <title>ASF JIRA</title>
    <link>https://issues.apache.org/jira</link>
    <description>This file is an XML representation of an issue</description>
    <language>en-uk</language>    <build-info>
        <version>6.3.4</version>
        <build-number>6332</build-number>
        <build-date>15-08-2014</build-date>
    </build-info>

<item>
            <title>[DERBY-6058] 2 ClassNotFoundException failures on trunk with ibm&apos;s weme6.2 since build 1433263</title>
                <link>https://issues.apache.org/jira/browse/DERBY-6058</link>
                <project id="10594" key="DERBY">Derby</project>
                    <description>&lt;p&gt;Since the build of 1433263 on trunk, we see the following test failures:&lt;/p&gt;

&lt;p&gt;1) noSpecialCollation(org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest)java.sql.SQLException: The class &apos;org.apache.derbyTesting.functionTests.tests.lang.VarargsRoutines&apos; does not exist or is inaccessible. This can happen if the class is not public.&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedPreparedStatement.&amp;lt;init&amp;gt;(Unknown Source)&lt;br/&gt;
	at org.apache.derby.jdbc.Driver169.newEmbedPreparedStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.prepareStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.prepareStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.prepareStatement(BaseJDBCTestCase.java:217)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.assertResults(TableFunctionTest.java:2579)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.assertResults(TableFunctionTest.java:2568)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.derby_6040(TableFunctionTest.java:2370)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.miscBugs(TableFunctionTest.java:1936)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.tableFunctionTest(TableFunctionTest.java:1005)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.noSpecialCollation(TableFunctionTest.java:971)&lt;br/&gt;
	at java.lang.reflect.AccessibleObject.invokeV(AccessibleObject.java:195)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:117)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.runBareOverridable(BaseJDBCTestCase.java:424)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.runBare(BaseJDBCTestCase.java:441)&lt;br/&gt;
Caused by: java.sql.SQLException: Java exception: &apos;org.apache.derbyTesting.functionTests.tests.lang.VarargsRoutines : JVMCFRE003 bad major version; class=org/apache/derbyTesting/functionTests/tests/lang/VarargsRoutines, offset=6: java.lang.ClassNotFoundException&apos;.&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.javaException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)&lt;br/&gt;
	... 37 more&lt;br/&gt;
Caused by: java.lang.ClassNotFoundException: org.apache.derbyTesting.functionTests.tests.lang.VarargsRoutines : JVMCFRE003 bad major version; class=org/apache/derbyTesting/functionTests/tests/lang/VarargsRoutines, offset=6&lt;br/&gt;
	at org.apache.derby.impl.services.reflect.DatabaseClasses.loadApplicationClass(Unknown Source)&lt;br/&gt;
	at org.apache.derby.iapi.services.loader.ClassInspector.getClass(Unknown Source)&lt;br/&gt;
	at org.apache.derby.iapi.services.loader.ClassInspector.accessible(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.QueryTreeNode.verifyClassExist(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.StaticMethodCallNode.bindExpression(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.FromVTI.bindVTITables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.FromList.bindTables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.SelectNode.bindNonVTITables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.DMLStatementNode.bindTables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.DMLStatementNode.bind(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.CursorNode.bindStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)&lt;br/&gt;
	... 33 more&lt;br/&gt;
2) specialCollation(org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest)java.sql.SQLException: The class &apos;org.apache.derbyTesting.functionTests.tests.lang.VarargsRoutines&apos; does not exist or is inaccessible. This can happen if the class is not public.&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedPreparedStatement.&amp;lt;init&amp;gt;(Unknown Source)&lt;br/&gt;
	at org.apache.derby.jdbc.Driver169.newEmbedPreparedStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.prepareStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.EmbedConnection.prepareStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.prepareStatement(BaseJDBCTestCase.java:217)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.assertResults(TableFunctionTest.java:2579)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.assertResults(TableFunctionTest.java:2568)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.derby_6040(TableFunctionTest.java:2370)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.miscBugs(TableFunctionTest.java:1936)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.tableFunctionTest(TableFunctionTest.java:1005)&lt;br/&gt;
	at org.apache.derbyTesting.functionTests.tests.lang.TableFunctionTest.specialCollation(TableFunctionTest.java:981)&lt;br/&gt;
	at java.lang.reflect.AccessibleObject.invokeV(AccessibleObject.java:195)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestCase.runBare(BaseTestCase.java:117)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.runBareOverridable(BaseJDBCTestCase.java:424)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseJDBCTestCase.runBare(BaseJDBCTestCase.java:441)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
	at org.apache.derbyTesting.junit.BaseTestSetup.run(BaseTestSetup.java:57)&lt;br/&gt;
	at junit.extensions.TestDecorator.basicRun(TestDecorator.java:24)&lt;br/&gt;
	at junit.extensions.TestSetup$1.protect(TestSetup.java:21)&lt;br/&gt;
	at junit.extensions.TestSetup.run(TestSetup.java:25)&lt;br/&gt;
Caused by: java.sql.SQLException: Java exception: &apos;org.apache.derbyTesting.functionTests.tests.lang.VarargsRoutines : JVMCFRE003 bad major version; class=org/apache/derbyTesting/functionTests/tests/lang/VarargsRoutines, offset=6: java.lang.ClassNotFoundException&apos;.&lt;br/&gt;
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.Util.javaException(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)&lt;br/&gt;
	... 52 more&lt;br/&gt;
Caused by: java.lang.ClassNotFoundException: org.apache.derbyTesting.functionTests.tests.lang.VarargsRoutines : JVMCFRE003 bad major version; class=org/apache/derbyTesting/functionTests/tests/lang/VarargsRoutines, offset=6&lt;br/&gt;
	at org.apache.derby.impl.services.reflect.DatabaseClasses.loadApplicationClass(Unknown Source)&lt;br/&gt;
	at org.apache.derby.iapi.services.loader.ClassInspector.getClass(Unknown Source)&lt;br/&gt;
	at org.apache.derby.iapi.services.loader.ClassInspector.accessible(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.QueryTreeNode.verifyClassExist(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.StaticMethodCallNode.bindExpression(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.FromVTI.bindVTITables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.FromList.bindTables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.SelectNode.bindNonVTITables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.DMLStatementNode.bindTables(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.DMLStatementNode.bind(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.compile.CursorNode.bindStatement(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)&lt;br/&gt;
	at org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)&lt;br/&gt;
	... 48 more&lt;/p&gt;

&lt;p&gt;Things were fine with build 1432788.&lt;br/&gt;
Things are also fine with ibm 1.5, 1.6, 1.7.&lt;/p&gt;

&lt;p&gt;See also: &lt;a href=&quot;http://people.apache.org/~myrnavl/derby_test_results/main/windows/testSummary-1433263.html&quot; class=&quot;external-link&quot; rel=&quot;nofollow&quot;&gt;http://people.apache.org/~myrnavl/derby_test_results/main/windows/testSummary-1433263.html&lt;/a&gt;&lt;/p&gt;</description>
                <environment>windows xp with ibm&amp;#39;s cdc/foundation implementation weme6.2.</environment>
        <key id="12629601">DERBY-6058</key>
            <summary>2 ClassNotFoundException failures on trunk with ibm&apos;s weme6.2 since build 1433263</summary>
                <type id="1" iconUrl="https://issues.apache.org/jira/images/icons/issuetypes/bug.png">Bug</type>
                                            <priority id="3" iconUrl="https://issues.apache.org/jira/images/icons/priorities/major.png">Major</priority>
                        <status id="6" iconUrl="https://issues.apache.org/jira/images/icons/statuses/closed.png" description="The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.">Closed</status>
                    <statusCategory id="3" key="done" colorName="green"/>
                                    <resolution id="1">Fixed</resolution>
                                        <assignee username="rhillegas">Rick Hillegas</assignee>
                                    <reporter username="myrna">Myrna van Lunteren</reporter>
                        <labels>
                    </labels>
                <created>Mon, 28 Jan 2013 22:57:49 +0000</created>
                <updated>Wed, 3 Sep 2014 09:31:38 +0100</updated>
                            <resolved>Tue, 29 Jan 2013 13:40:36 +0000</resolved>
                                    <version>10.10.1.1</version>
                                    <fixVersion>10.10.1.1</fixVersion>
                                    <component>Test</component>
                        <due></due>
                            <votes>0</votes>
                                    <watches>4</watches>
                                                                <comments>
                            <comment id="13565357" author="rhillegas" created="Tue, 29 Jan 2013 13:33:09 +0000"  >&lt;p&gt;Linking to &lt;a href=&quot;https://issues.apache.org/jira/browse/DERBY-6040&quot; title=&quot;Incorrect row order returned for an ORDER BY on a join of two table functions&quot; class=&quot;issue-link&quot; data-issue-key=&quot;DERBY-6040&quot;&gt;&lt;del&gt;DERBY-6040&lt;/del&gt;&lt;/a&gt; because the failing test was introduced by work on that issue.&lt;/p&gt;</comment>
                            <comment id="13565363" author="rhillegas" created="Tue, 29 Jan 2013 13:40:19 +0000"  >&lt;p&gt;Thanks for finding this problem, Myrna. Attaching derby-6058-01-aa-disableOnJSR169.diff. This patch disables the failing test on JSR169. Committed at subversion revision 1439889.&lt;/p&gt;

&lt;p&gt;The failing test uses a varargs table function which only runs on VM level 5 and above.&lt;/p&gt;

&lt;p&gt;Before applying this patch, the test failed for me on OJEC. After applying the patch, the test succeeds on OJEC.&lt;/p&gt;

&lt;p&gt;Touches the following file:&lt;/p&gt;

&lt;p&gt;M       java/testing/org/apache/derbyTesting/functionTests/tests/lang/TableFunctionTest.java&lt;/p&gt;</comment>
                            <comment id="14119613" author="knutanders" created="Wed, 3 Sep 2014 09:31:38 +0100"  >&lt;p&gt;&lt;span class=&quot;error&quot;&gt;&amp;#91;bulk update&amp;#93;&lt;/span&gt; Close all resolved issues that haven&apos;t been updated for more than one year.&lt;/p&gt;</comment>
                    </comments>
                <issuelinks>
                            <issuelinktype id="10030">
                    <name>Reference</name>
                                            <outwardlinks description="relates to">
                                        <issuelink>
            <issuekey id="12627022">DERBY-6040</issuekey>
        </issuelink>
                            </outwardlinks>
                                                        </issuelinktype>
                    </issuelinks>
                <attachments>
                            <attachment id="12566953" name="derby-6058-01-aa-disableOnJSR169.diff" size="723" author="rhillegas" created="Tue, 29 Jan 2013 13:40:19 +0000"/>
                    </attachments>
                <subtasks>
                    </subtasks>
                <customfields>
                                                <customfield id="customfield_12310310" key="com.atlassian.jira.toolkit:attachments">
                        <customfieldname>Attachment count</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>1.0</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        <customfield id="customfield_12310220" key="com.atlassian.jira.ext.charting:firstresponsedate">
                        <customfieldname>Date of First Response</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>Tue, 29 Jan 2013 13:33:09 +0000</customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                                                                                                                                                                    <customfield id="customfield_12310420" key="com.pyxis.greenhopper.jira:gh-global-rank">
                        <customfieldname>Global Rank</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>310097</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                <customfield id="customfield_12311820" key="com.pyxis.greenhopper.jira:gh-lexo-rank">
                        <customfieldname>Rank</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>2|hzbd8f:</customfieldvalue>

                        </customfieldvalues>
                    </customfield>
                                                                <customfield id="customfield_12310920" key="com.pyxis.greenhopper.jira:gh-global-rank">
                        <customfieldname>Rank (Obsolete)</customfieldname>
                        <customfieldvalues>
                            <customfieldvalue>310442</customfieldvalue>
                        </customfieldvalues>
                    </customfield>
                                                                                                                                                                                                                                        <customfield id="customfield_12310222" key="com.atlassian.jira.ext.charting:timeinstatus">
                        <customfieldname>Time in Status</customfieldname>
                        <customfieldvalues>
                            
                        </customfieldvalues>
                    </customfield>
                                                                                            </customfields>
    </item>
</channel>
</rss>